numpy/random/_mt19937.pyx

```diff
@@ -214,7 +214,7 @@ cdef class MT19937(BitGenerator):
 
         Returns a new bit generator with the state jumped
 
-        The state of the returned big generator is jumped as-if
+        The state of the returned bit generator is jumped as-if
         2**(128 * jumps) random numbers have been generated.
```

Comment:
This seems suspiciously similar to #21352. @rkern, @bashtage is the formula wrong here too?